server/server.go

```diff
@@ -0,0 +1,384 @@
+package server
```

Comment:
This new package includes a lot of the logic that used to live in the zeno command. Pulling it out into its own package makes it easier to test as well as embed.